### PR TITLE
Resolves #114 [Doc] add links to frontend files

### LINKED ISSUE #114: [Doc] add link to components file in the frontend documentation

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Fix #114

### CODE REVIEW

File: doc/frontend/frontend.md [hunk 1]
```diff
@@ -15,21 +15,34 @@
 - activity-navbar :
   > display description room and if a user wrinting in the input
 
+  ![html](../../logos/html.png) [html file](front/activity_bar.html)
+
```
Comment (Draeggon) [inline doc/frontend/frontend.md:19]: the link to the html file doesn't work
it need to go back 2 file to be in  scratchy directory like:
../../front/activity_bar.html
and same for every link

**Review by Draeggon — CHANGES_REQUESTED** [review]

File: doc/frontend/frontend.md (hunk 1 above)
Comment (yannis-mlgrn) [reply doc/frontend/frontend.md:26]: done here cfb273a04b630a4296bdac87f4fca0a0723ecbf9

Comment (Draeggon) [reply doc/frontend/frontend.md:26]: :facepalm:  you haven't fixed anything

Comment (yannis-mlgrn) [reply doc/frontend/frontend.md:26]: really done here 3c73110b3e18d1aaee21de9766b1c34c4e85b98e

Comment (yannis-mlgrn) [reply doc/frontend/frontend.md:26]: V3 for problem on directory link

**Review by Draeggon — APPROVED** [review]

**Review by Draeggon — APPROVED** [review]

**Pull request merged**